requirements.txt

```diff
@@ -1,4 +1,4 @@
 scipy>=1.2.0
 numpy
-pandas>=1.1.0
+pandas>=1.5.0
```

Comment:
Note: This also changes the Python version support to `Python>=3.8`.

Reply:
This is a shame, as I wanted to keep 3.6 (since that's the default Python on rhel8 a-la CSD3), but I agree that if pandas is now requiring 3.8, then we will have to as well.